### PR TITLE
[Relay] Create header file for realize.cc

### PR DESCRIPTION
**Pull request by jonathansparling**
Closes https://github.com/apache/tvm/issues/11092

### CODE REVIEW

File: src/relay/quantize/realize.h [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#ifndef TVM_RELAY_QUANTIZE_REALIZE_H_
+#define TVM_RELAY_QUANTIZE_REALIZE_H_
+
+#include <tvm/relay/transform.h>
+
+namespace tvm {
+namespace relay {
+namespace quantize {
+
+class QRealizeExpr;
```
Comment (AndrewZhaoLuo) [inline src/relay/quantize/realize.h:36]: Do we need these declarations as we define things below?

**Review by AndrewZhaoLuo — APPROVED** [review]
LGTM, one q

**Comment by AndrewZhaoLuo** [comment]
Also CI is sometimes flaky, in that case you should push a new commit to it (an empty commit will work)

File: src/relay/quantize/realize.h (hunk 1 above)
Comment (jonathansparling) [reply src/relay/quantize/realize.h]: Nope, I removed them in the fourth commit.

**Pull request merged**